mail_notification_custom_subject/tests/test_mail_notification_custom_subject.py

```diff
@@ -19,21 +19,19 @@ def setUpClass(cls):
             ]
         )
         cls.admin = new_test_user(cls.env, "boss", "base.group_system")
+        new_test_user(cls.env, "worker")
```

Comment:
Avoid user name collapse:

```suggestion
        new_test_user(cls.env, "worker_custom_subject")
```